docs/excel/excel-add-ins-conditional-formatting.md

```diff
@@ -0,0 +1,305 @@
+conditionalFormat.preset.format.font.color = "white";
+conditionalFormat.preset.rule = {
+     criterion: Excel.ConditionalFormatPresetCriterion.oneStdDevAboveAverage
+};
+
+await context.sync();
+```
+
+### [Text comparison](https://docs.microsoft.com/javascript/api/excel/excel.textconditionalformat)
+
+Text comparison formatting uses string comparisons as the condition. The `rule` property is a [ConditionalTextComparisonRule](https://docs.microsoft.com/javascript/api/excel/excel.conditionaltextcomparisonrule) defining a string to compare with the cell and an operator to specify the type of comparison. The following example formats the font color red when a cell’s text contains “Delayed”.
+
+![A range with cells containing "Delayed" in red.](../images/excel-conditional-format-text.png)
+
+```typescript
+const sheet = context.workbook.worksheets.getItem("Sample");
+const range = sheet.getRange("B16:D18");
+const conditionalFormat = range.conditionalFormats.add(
+     Excel.ConditionalFormatType.containsText
+);
+
+// color the font of every cell containing “Delayed”
+conditionalFormat.textComparison.format.font.color = "red";
+conditionalFormat.textComparison.rule = {
+     operator: Excel.ConditionalTextOperator.contains,
+     text: "Delayed"
+};
+
+await context.sync();
+```
+
+### [Top/bottom](https://docs.microsoft.com/javascript/api/excel/excel.TopBottomconditionalformat)
+
+Top/bottom formatting applies a format to the highest or lowest values in a range. Whether the condition is based on the highest or lowest, as well as whether the evaluation is ranked or percentage-based, is set by the `rule` property of type [ConditionalTopBottomRule](https://docs.microsoft.com/javascript/api/excel/excel.conditionaltopbottomrule). The following example applies a green highlight to the highest value cell in the range.
+
+![A range with the highest number highlighted in green.](../images/excel-conditional-format-topbottom.png)
+
+```typescript
+const sheet = context.workbook.worksheets.getItem("Sample");
+const range = sheet.getRange("B21:E23");
+const conditionalFormat = range.conditionalFormats.add(
+     Excel.ConditionalFormatType.topBottom
+);
+
+// for the highest valued cell in the range, make the background green
+conditionalFormat.topBottom.format.fill.color = "green"
+conditionalFormat.topBottom.rule = { rank: 1, type: "TopItems"}
+
+await context.sync();
+```
+
+## Multiple Formats and Priority
+
+Ranges can have multiple conditional formats affecting them. If the formats have conflicting elements, such as differing font colors, only one format applies that particular element. Precedence is defined by the `ConditionalFormat.priority` property. Priority is a number (equal to the index in the `ConditionalFormatCollection`) and can be set when creating the format. The smaller the priority value, the higher priority that format will have (a conflict between a priority-0 and a priority-1 resolves in favor of the 0).
```

Comment:
Consider rewording opening sentence something along these lines:

You can apply multiple conditional formatting rules to a range.

Reply:
Is it necessary to include "(a conflict between a priority-0 and a priority-1 resolves in favor of the 0)" at the end of the final sentence?  If so, I'd suggest rewording for clarity...seems confusing as currently written.